pandas/tests/resample/test_datetime_index.py

```diff
@@ -789,11 +796,35 @@ def test_resample_base():
     rng = date_range("1/1/2000 00:00:00", "1/1/2000 02:00", freq="s")
     ts = Series(np.random.randn(len(rng)), index=rng)
 
-    resampled = ts.resample("5min", base=2).mean()
+    with tm.assert_produces_warning(FutureWarning):
+        resampled = ts.resample("5min", base=2).mean()
     exp_rng = date_range("12/31/1999 23:57:00", "1/1/2000 01:57", freq="5min")
     tm.assert_index_equal(resampled.index, exp_rng)
 
 
+def test_resample_offset():
```

Comment:
add the issue number as a comment

Reply:
Done in the last commit for all tests added.